sdk/python/feast/feature_service.py

```diff
@@ -19,21 +19,25 @@
 
 class FeatureService:
     """
-    A feature service is a logical grouping of features for retrieval (training or serving).
-    The features grouped by a feature service may come from any number of feature views.
-
-    Args:
-        name: Unique name of the feature service.
-        features: A list of Features that are grouped as part of this FeatureService.
-            The list may contain Feature Views, Feature Tables, or a subset of either.
-        tags (optional): A dictionary of key-value pairs used for organizing Feature
-            Services.
+    A feature service defines a logical group of features from one or more feature views.
+    This group of features can be retrieved together during training or serving.
+
+    Attributes:
+        name: The unique name of the feature service.
+        feature_view_projections: A list containing feature views and feature view
+            projections, representing the features in the feature service.
+        description: A human-readable description.
+        tags: A dictionary of key-value pairs to store arbitrary metadata.
+        owner: The owner of the feature service.
```

Comment:
Should we give more details here? Should it be an email? Is this owner already registered elsewhere in the feature store? Can it be any string?

Reply:
Sure, added a note about email.